lib/rubocop/cop/style/empty_line_after_guard_clause.rb

```diff
@@ -68,6 +68,10 @@ def next_line_empty?(node)
           processed_source[node.last_line].blank?
         end
 
+        def next_line_rescue_or_ensure?(node)
```

Comment:
Method name seems a bit strange to me. It talks about "next line", but receiving node and checks its type. I think it's better either rename method or pass node instead of its parent.

Reply:
Thanks for the review. Since I cannot hit upon a good method name, I changed to pass the node 💦

Reply:
I also failed to find better name, so I just wrote "rename" 😆